src/preact/base-element.js

```diff
@@ -625,7 +710,9 @@ function shouldMutationForNodeListBeRerendered(nodeList) {
       // Ignore service elements, e.g. `<i-amphtml-svc>` or
       // `<x slot="i-amphtml-svc">`.
       if (
-        startsWith(node.tagName, 'I-AMPHTML') ||
+        node[RENDERED_PROP] ||
+        node.hasAttribute(RENDERED_PROP) ||
```

Comment:
Do we ever set an attribute with this name?

Reply:
I can remove this for now. The main place where this could pop up is the SSR case. Preact deals with this as following: if a prop exits on the node instance (e.g. `'x' in node`) then Preact assigns it as a property (`node['x'] = value`), otherwise as an attribute (`node.setAttribute('x', value)`). So for now it's probably a bit overprotective. Would you rather this removed?

Reply:
Removed. We'll put it back if really needed later.